### DISCUSSION TITLE
Setup wizard

### DISCUSSION THREAD

**Discussion by AlfHou** [jellyfin/jellyfin-vue#269] [Features]
First time users need to setup the jellyfin server by adding a user and setting up the libraries.

**Comment by AlfHou** [comment]
This is required for _Feature parity with Jellyfin Web_

**Comment by heyhippari** [comment]
If someone is interested in working on this in the future, this is one of the rare features where we can provide a UI concept.

[Setup wizard.pdf](https://github.com/jellyfin/jellyfin-vue/files/5529361/Setup.wizard.pdf)

Do note that, compared to the current web, some screens have their position modified (The library language now comes before the library setup).
